examples/skr/aks/README.md

```diff
@@ -56,9 +56,22 @@ West Europe: sharedweu.weu.attest.azure.net
 This is a one time effort.
 Once the key is created, it can be used to protect as many secrets as desired.
 
-The `setup-key-mhsm.sh` script creates a private/public key pair in the keyvault and a key release policy with the file name `keyname-release-policy.json`.
-The public key is then downloaded as `keyname-pub.pem` and a key info file is generated.
+The `setup-key-mhsm.sh` script creates a private/public key pair in the keyvault and a key release policy with the file name `<keyname>-release-policy.json`.
```

Comment:
since the json is labeled "KMS", is it worth saying this setup is for Azure mHSM but other AKV can also be used?

Reply:
Yeah, that's my issue with us not being consistent on the naming. All of our setup is for AKV/MAA and so the names in the files are for that (with the exception of the AASP code), but I was under the impression that other kms/attestation endpoints were usable, so it makes more sense to update variables to be generic and specify that the setup scripts assume the usage of Azure kms/attestation.